lib/gli/commands/help_modules/command_help_format.rb

```diff
@@ -13,38 +13,38 @@ def initialize(command,app,sorter,synopsis_formatter_class,wrapper_class=TextWra
         end
 
         def format
-          command_wrapper      = @wrapper_class.new(Terminal.instance.size[0],4 + @command.name.to_s.size + 3)
-          wrapper              = @wrapper_class.new(Terminal.instance.size[0],4)
+          @command_wrapper      = @wrapper_class.new(Terminal.instance.size[0],4 + @command.name.to_s.size + 3)
```

Comment:
I've done the CleanRoom approach as well, but using `OpenStruct`:

```ruby
require 'ostruct'

class CleanRoom < OpenStruct
  def render_erb template
    ERB.new(template).result(binding)
  end
end
```

Not sure if this makes it more acceptable or less.